src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2142,9 +2155,16 @@ private void CheckForLatestRender(object obj)
 
         public DynamoViewModel ViewModel { get { return this; } }
 
-        #endregion
-
+        public IBrandingResourceProvider BrandingResourceProvider { get; private set; }
 
+        public ImageSource AboutBoxIcon
+        {
+            get
+            {
+                return BrandingResourceProvider.GetImageSource(ResourceName.AboutBoxLogo);
+            }
+        }
 
+        #endregion
```

Comment:
Sorry for not making myself clear, it is only the file name, `DynamoViewModel.Branding.cs`. There is nothing to change about this class, just the code location (in a separate file which is easier to maintain and reason about).